libs/sdk-core/src/breez_services.rs

```diff
@@ -325,6 +325,16 @@ impl BreezServices {
             {
                 Ok(res) => Some(res),
                 Err(e) => {
+                    if e.to_string().contains("missing balance") {
+                        debug!(
+                            "trampoline payment failed due to insufficient balance: {:?}",
+                            e
+                        );
+                        return Err(SendPaymentError::InsufficientBalance {
+                            err: "Insufficient balance for trampoline payment".into(),
```

Comment:
My only nit here would be it produces an error "Insufficient balance: Insufficient balance for trampoline payment". Maybe?
```suggestion
                            err: "Tampoline payment failed".into(),
```

Reply:
Agreed, done!